sdk/src/stream_builder/iggy_stream.rs

```diff
@@ -0,0 +1,78 @@
+use crate::client::SystemClient;
+use crate::clients::client::IggyClient;
+use crate::clients::consumer::IggyConsumer;
+use crate::clients::producer::IggyProducer;
+use crate::error::IggyError;
+use crate::stream_builder::build;
+use crate::stream_config::IggyStreamConfig;
+use tracing::info;
+
+#[derive(Debug, Default, Clone, Eq, PartialEq)]
+pub struct IggyStream {}
```

Comment:
Since this struct has no fields, it could be just `pub struct IggyStream` without the `{}`